R/bin.R

```diff
@@ -1,6 +1,7 @@
 bins <- function(breaks, closed = "right",
-                 fuzz = 1e-08 * stats::median(diff(breaks))) {
+                 fuzz = NULL) {
   check_numeric(breaks)
+  fuzz <- fuzz %||% 1e-08 * stats::median(diff(breaks[is.finite(breaks)]))
```

Comment:
what happens if there is only one finite break?

Reply:
Currently, it throws an error. I guess a not unreasonable approach would be to set the fuzz to a very small number?

Reply:
yeah, I'd be fine with that, but we should not throw error at least